pkg/coscheduling/apis/podgroup/v1alpha1/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Move the apis from  pkg/coscheduling/apis to pkg/apis/

Reply:
But  I thinks this apis  are only for coscheduling

Reply:
PodGroup is a universal concept. It maybe used by other plugins.